code/__DEFINES/food.dm

```diff
@@ -177,6 +177,14 @@ DEFINE_BITFIELD(food_flags, list(
 
 #define BLACKBOX_LOG_FOOD_MADE(food) SSblackbox.record_feedback("tally", "food_made", 1, food)
 
+/// Point water boils at
+#define WATER_BOILING_POINT 373
```

Comment:
Make this something like

```suggestion
#define WATER_BOILING_POINT T0C + 100
```

Where T0C is 0 degrees celsius (273.15 Kelvin), and we can derive that number from there. Much more sciency!